### ISSUE TITLE
Offering $10,000.00-$20,000.00 USD for Upgrades to Project (COD: Black Ops 2)

### ISSUE THREAD

**Issue by creed59** [Laupetin/OpenAssetTools#235]
Hello Laupetin:
I am offering to contribute $10k-$20k USD to your OAT project in return for the following modifications to be made to it:

                                                     **For Black Ops 2 only**
- A .MAP file to .D3DBSP converter, some kind of BSP-generating tool
- A form of CoDRadiant as part of the toolset. [https://callofduty.fandom.com/wiki/Radiant](url)
- The ability to patch the linker to link BSP assets
- A mod launcher to interface with the linker, with a UI as an executable program ( to link bsp assets and more things):
basically everything that is featured in this photo (My Mods, Asset Manager, IWI Tool, Ipak Explorer, Havok Utilities etc.):
![bo2-tools](https://github.com/user-attachments/assets/0290e3fe-b571-40cc-8296-683c16677f48)
The sooner you can work on it, the more I'll offer.
Thanks for reading.

**Comment by Laupetin** [comment]
Hey there :) thanks for the offer, I'm glad you seem to be interested in the project.

The thing is that this is entirely a hobby project for me and I dont want to take money for it or make money from it.
So I am not planning to take any donations or monetary contributions, sorry 😅
It would change the way I perceive and work on the project which I dont really want nor have the capacity for.

The good aspect though is that basically all of what you mentioned is part of what I am planning to add at some point in the future.
It may take a bit though, I do not have any specific plans for Radiant stuff for example.
A UI for the tools is planned however and I am currently experimenting a bit for it.

If you have specific feature ideas for the different parts of your offer, feel free to open an issue for a feature request.

I hope you understand :D feel free to reach out to me on discord if you want to talk about ideas.
